tools/UpdatePaths/Scripts/76296_working_mecha_nuke.txt

```diff
@@ -0,0 +1,3 @@
+# In this PR, the working mecha subtype was removed.
+
+/obj/vehicle/sealed/mecha/working/@SUBTYPES : /obj/vehicle/sealed/mecha/@SUBTYPES
```

Comment:
```suggestion
/obj/vehicle/sealed/mecha/working/@SUBTYPES : /obj/vehicle/sealed/mecha/@SUBTYPES{@OLD}
```

you accidentally nuked any and all var edits that people may have done to these objects, can you kindly revert map changes and re-run this script? thanks

Reply:
aw fuck